File: nodecore-spv/src/main/java/veriblock/model/LedgerProofStatus.java
package veriblock.model;

public enum LedgerProofStatus {
    ADDRESS_EXISTS,
    ADDRESS_DOES_NOT_EXIST,
    ADDRESS_IS_INVALID;


    public static LedgerProofStatus getByOrdinal(int ordinal){
        return LedgerProofStatus.values()[ordinal];
    }

    public boolean isExists(){
        return this == LedgerProofStatus.ADDRESS_EXISTS;
    }
}

File: nodecore-spv/src/main/java/veriblock/model/LedgerProofStatus.kt
package veriblock.model

enum class LedgerProofStatus {
    ADDRESS_EXISTS,
    ADDRESS_DOES_NOT_EXIST,
    ADDRESS_IS_INVALID;

    fun exists(): Boolean =
        this == ADDRESS_EXISTS

    companion object {
        fun getByOrdinal(ordinal: Int): LedgerProofStatus {
            return values()[ordinal]
        }
    }
}
